website/screens/components/button/usage/ButtonUsagePage.tsx

```diff
@@ -91,15 +92,15 @@ const sections = [
     content: (
       <>
         <DxcParagraph>
-          Any icon can be used in Halstack Design System, so it is apt to the
-          user to choose between one of the multiple options that are offered
-          through internet but it is recommended to use the same library of
-          icons along the application to keeping concistency.
+          Any icon can be used in Halstack Design System, so it is up to the
+          user to choose which one suits its use case better. But, it is
+          recommended to use the same library of icons throughout the
+          application to keep consistency.
         </DxcParagraph>
         <DxcBulletedList>
           <DxcBulletedList.Item>
-            Icon can go before or after the text with a separation of 8 pixels
-            respecting the main button&#39;s text.
+            The icon can go before or after the text with a separation of 8
+            pixels respecting the main button's text.
```

Comment:
We should maintain &#39; instead of "'".

Reply:
For the moment, we are going to keep this until we know more about this issue.